app/models/qa/authorities/local/find_works.rb

```diff
@@ -0,0 +1,23 @@
+module Qa::Authorities
+  class Local::FindWorks < ActiveFedora::Base
+    attr_reader :q
+
+    def initialize(q)
+      @q = q
```

Comment:
Why are you assigning `@q`, or even defining `initialize`? This should be handled by the base.